.stylua.toml

```diff
@@ -0,0 +1,6 @@
+column_width = 120
+line_endings = "Unix"
+indent_type = "Spaces"
+indent_width = 2
+quote_style = "AutoPreferSingle"
+no_call_parentheses = false
```

Comment:
`column_width = 120`, `line_endings = "Unix"` and `no_call_parentheses = false` are [defaults](https://github.com/JohnnyMorganz/StyLua#finding-the-configuration) in stylua. You may omit it as suggested in the readme.

Reply:
-1, defaults can change and it's nice to be explicit